augur/market_info.py

```diff
@@ -69,48 +70,48 @@ def universe(self):
     return self._universe
 
   @property
-  def marketType(self):
+  def market_type(self):
     '''Type of Market ("yesNo", "categorical", or "scalar").
```

Comment:
Make this an enum, that is... define a class:

```
class Type(enum.Enum):
  YES_NO = auto()
   ...
```

This enum should be defined within MarketInfo so that it may be used as follows:

```MarketInfo.Type.SCALAR```

Reply:
Update the documentation to reflect the use of the ```Type``` enum.